.github/actions/setup-kurtosis-type1/action.yml

```diff
@@ -15,17 +15,45 @@ runs:
       uses: actions/checkout@v4
       with:
         repository: 0xPolygon/kurtosis-cdk
-        ref: v0.4.4
+        ref: v0.4.6
         path: kurtosis-cdk
 
     - name: Install Kurtosis CDK tools
       if: env.SKIP_ENCLAVE_SETUP != 'true'
-      uses: ./kurtosis-cdk/.github/actions/setup-kurtosis-cdk
+      uses: ./kurtosis-cdk/.github/actions/kurtosis-pre-run
 
     - name: Install Foundry
       if: env.SKIP_ENCLAVE_SETUP != 'true'
       uses: foundry-rs/foundry-toolchain@v1
 
+    - name: Install polycli
+      shell: bash
+      run: |
+        tmp_dir=$(mktemp -d) && curl -L https://github.com/0xPolygon/polygon-cli/releases/download/v0.1.48/polycli_v0.1.48_linux_amd64.tar.gz | tar -xz -C "$tmp_dir" && mv "$tmp_dir"/* /usr/local/bin/polycli && rm -rf "$tmp_dir"
+        sudo chmod +x /usr/local/bin/polycli
+        /usr/local/bin/polycli version
+
+    - name: Install yq
+      shell: bash
+      run: |
+        sudo curl -L https://github.com/mikefarah/yq/releases/download/v4.44.2/yq_linux_amd64 -o /usr/local/bin/yq
+        sudo chmod +x /usr/local/bin/yq
+        /usr/local/bin/yq --version
+
+    # Cleanup before running Kurtosis to have more space. Those dirs are claimed to be universal and safe to remove
```

Comment:
I've made a separate freeup-disk-space action for it. Will it work for you? But I am bit worried that pruning docker containers will actually remove the ones installed by kurtosis-pre-run, because kurtosis is not started at the moment currently.
I think I tried to use freeup-disk-space action and removed it use because of this failure. Not sure why such cleanup here wont clean up the kurtosis images.

Reply:
For the amount of code there I'm happy to leave this in as it is rather than using another action.